Analytical_Engine/Query/Graph/EntityNeighbourhood.cs

```diff
@@ -46,6 +46,12 @@ public static partial class Query
         [Output("graphs", "The collection of sub Graphs found in the input Graph.")]
         public static List<Graph> EntityNeighbourhood(this Graph graph, RelationDirection relationDirection = RelationDirection.Forwards)
         {
+            if (graph == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the entity neighbourhood of a null graph.");
+                return null;
```

Comment:
Why not return a list to follow the pattern?
```suggestion
                return new List<Graph>();
```